src/GovUk.Education.ExploreEducationStatistics.Data.Services.Tests/SubjectMetaServiceTests.cs

```diff
@@ -509,12 +499,422 @@ await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
             }
         }
 
-        private static IFeatureManager DefaultFeatureManager()
+        [Fact]
+        public async Task GetSubjectMeta_LocationsAreDeduplicated_Flat()
+        {
+            var release = new Release();
+            var subject = new Subject();
+
+            var releaseSubject = new ReleaseSubject
+            {
+                Release = release,
+                Subject = subject
+            };
+
+            var locations = new Dictionary<GeographicLevel, List<LocationAttributeNode>>
+            {
+                {
+                    GeographicLevel.LocalAuthority,
+                    // Region-LA hierarchy in the LA level data
```

Comment:
I think this comment is wrong as there is no hierarchy.

Reply:
Removed!